buildSrc/src/main/kotlin/kotlin-conventions.gradle.kts

```diff
@@ -32,6 +34,81 @@ kotlin {
     }
 }
 
+val prepareReadmeForDokka by tasks.registering {
```

Comment:
Could we move it in separate plugin like `dokka-conventions`?

Reply:
Good idea, done!